index.js

```diff
@@ -79,15 +83,17 @@ async function main() {
     const DEFAULT_BRANCH = default_branch;
     core.debug(`DEFAULT_BRANCH: ${DEFAULT_BRANCH}`);
 
-    const { hasChanges } = await getLocalChanges(inputs.path);
+    if (inputs.path) {
+      core.debug(`Changing directory to ${inputs.path}`);
+      process.chdir(inputs.path);
```

Comment:
Is this a red flag? Are there other options I might use? I picked `process.chdir` because `runShellCommand` invokes a new terminal session each time it's run, and that defaults to the directory the code is being run from.